.ci.yaml

```diff
@@ -2511,7 +2511,7 @@ targets:
     properties:
       dependencies: >-
         [
-          {"dependency": "xcode", "version": "14a5294e"},
+          {"dependency": "xcode", "version": "14c18"},
```

Comment:
Instead of setting this at the builder level can we just set it once at the top `mac_benchmark`?

Reply:
Seems `Mac_benchmark flutter_tool_startup__macos` doesn't need an xcode. I guess it is okay for it to ask for an xcode if we move to the platform level?